release/templates/featuresenabled-azuredeploy.json

```diff
@@ -74,6 +74,7 @@
             "DicomFunctions__ConnectionName": "AzureWebJobsStorage",
             "DicomServer__Security__Enabled": "true",
             "DicomServer__Security__Authorization__Enabled": "true",
+            "DicomServer__Services__BlobMigration__FormatType": "2",
```

Comment:
Is it possible to have the configuration be the string representation of the enum `New`? I don't recall for sure off hand, but I believe that would work.